common/util.h

```diff
@@ -97,6 +97,12 @@ template<typename T> inline void SWAP(T &a, T &b) { T tmp = a; a = b; b = tmp; }
  */
 #define ARRAYEND(x) ((x) + ARRAYSIZE((x)))
 
+/**
+ * Convert statement to string via pre-processor
+ */
+#define STRINGIFY_X(s) #s
+#define STRINGIFY(s) STRINGIFY_X(s)
```

Comment:
I'm curious why this is chained instead of just doing #s directly

Reply:
this is because the pre-processor would otherwise just stringify the expression - not the result of the expression.

Reply:
I didn't know that. But if chained preprocessor macros execute expressions, then that means the change in common/system.h

```
#ifdef NDEBUG
// none debug build should rely on the system headers to define assert
#define scummvm_assert(cond, message) assert(cond);
```

This would be a chained call to assert, which means it might be actually evaluating the expression? I can test this out a bit later tonight.